sr_unix_cm/gtcm_bgn_net.c

```diff
@@ -140,6 +140,7 @@ int gtcm_bgn_net(omi_conn_ll *cll)
 		CLOSEFILE_RESET(fd, rc);	/* resets "fd" to FD_INVALID */
 		return save_errno;
 	}
+	freeaddrinfo(ai_ptr);
```

Comment:
There is a "return" at lines 126, 135 and 141 and I think we need to do the freeaddrinfo() there too.

Reply:
The 126 return is a getaddrinfo error return so don't think we need it there. Added to 135 and 141 though.

Reply:
Line 126 is a getnameinfo(), not a getaddrinfo() so I still think it needs a freeaddrinfo().

Reply:
OK, added.